### PR TITLE
Add URL(s) to status call

### LINKED ISSUE #199: Return URL property as part of app status call

### PR DESCRIPTION
**Pull request by ericbottard**
Fixes #199

### CODE REVIEW

**Comment by ilayaperumalg** [comment]
LGTM, merging.

File: src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppInstanceStatus.java [hunk 1]
```diff
@@ -87,6 +88,13 @@ public DeploymentState getState() {
 				attributes.put("metrics.machine.memory", String.format("%.1f%%", 100d * instanceDetail.getMemoryUsage() / instanceDetail.getMemoryQuota()));
 			}
 		}
+		List<String> urls = applicationDetail.getUrls();
+		if (!urls.isEmpty()) {
+			attributes.put("url", "http://" + urls.get(0));
+			for (int i = 0; i < urls.size() ; i++) {
+				attributes.put("url." + i, "http://" + urls.get(i));
```
Comment (ilayaperumalg) [inline src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppInstanceStatus.java]: does each instance of the app have a unique URL here?

Comment (ericbottard) [inline src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppInstanceStatus.java]: As discussed in Slack, no, and this is normal

Comment (ilayaperumalg) [inline src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppInstanceStatus.java]: The consensus here is that we need to show all the routes mapped for this app.

**Pull request merged**